README_JA.md

```diff
@@ -0,0 +1,479 @@
+Payday 2 シークレット
+---------------
+PD2のシークレットを解くための情報。
```

Comment:
I'd add a note to refer to the english doc. This doc potentially can get out of date.